services/bitbucket/bitbucket-last-commit.service.js

```diff
@@ -0,0 +1,77 @@
+import Joi from 'joi'
+import { age as ageColor } from '../color-formatters.js'
+import { BaseJsonService, NotFound, pathParam, queryParam } from '../index.js'
+import { formatDate } from '../text-formatters.js'
+
+const schema = Joi.object({
+  values: Joi.array().items({
+    date: Joi.string().isoDate().required(),
+  }),
+}).required()
+
+const queryParamSchema = Joi.object({
+  path: Joi.string().uri({ relativeOnly: true }),
+}).required()
+
+export default class BitbucketLastCommit extends BaseJsonService {
+  static category = 'activity'
+  static route = {
+    base: 'bitbucket/last-commit',
+    pattern: ':user/:repo/:branch*',
```

Comment:
Given it is a required param, this should be `branch+`. The `*` makes it optional.

```suggestion
    pattern: ':user/:repo/:branch+',
```